packages/core/src/odata/common/selectable/link.ts

```diff
@@ -1,9 +1,13 @@
 /* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
 
-import { Constructable } from '../constructable';
-import { EntityBase, EntityIdentifiable } from '../entity';
-import { Expandable } from '../expandable';
-import { Selectable } from './selectable';
+import { EntityBase, EntityIdentifiable, Constructable } from '../entity';
+import type { Expandable } from '../expandable';
+import type { Selectable } from './selectable';
+
+export type LinkType<
```

Comment:
Is is used?

Reply:
no also here I experimented and pushed it by accident.